### PR TITLE
Fix/add starter

### PR DESCRIPTION
**Pull request by RobertHebel**
Make starter repository part of this repo

### CODE REVIEW

File: example/src/pages/404.js [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import React from 'react';
+
+const NotFoundPage = () => (
+  <div>
+    <h1>NOT FOUND</h1>
+    <p>You just hit a route that doesn&#39;t exist... the sadness.</p>
```
Comment (piotr-s-brainhub) [inline example/src/pages/404.js:6]:
```suggestion
    <p>You just hit a route that doesn't exist... the sadness.</p>
```

Comment (piotr-s-brainhub) [reply example/src/pages/404.js:6]: would it work?

File: example/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    "@brainhubeu/gatsby-docs-kit": "^3.0.15",
+    "gatsby": "^2.20.22",
+    "gatsby-link": "^2.3.2"
+  },
+  "devDependencies": {
+    "babel-eslint": "^10.1.0",
+    "eslint": "^6.8.0",
+    "eslint-config-brainhub": "^1.12.0",
+    "gh-pages": "^2.2.0",
+    "react": "^16.13.1",
```
Comment (piotr-s-brainhub) [inline example/package.json:26]: why are `react` and `react-dom` dev dependencies?

File: example/README.md [hunk 3]
```diff
@@ -0,0 +1,41 @@
+# Gatsby Docs Kit Starter
+The default Brainhub static docs starter.
+
+[![CircleCI](https://circleci.com/gh/brainhubeu/gatsby-docs-kit-starter.svg?style=svg)](https://circleci.com/gh/brainhubeu/gatsby-docs-kit-starter)
```
Comment (piotr-s-brainhub) [inline example/README.md:4]: IMO these badges should be removed

File: tools/seed/my-first-docs.md [hunk 4]
```diff
@@ -24,6 +24,3 @@
     ```bash
     npm run deploy:gh
     ```
-
-    > See more [HERE](../getting-started/publishing.md)
```
Comment (piotr-s-brainhub) [inline tools/seed/my-first-docs.md:28]: why is this removed?

File: example/gatsby-config.js [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+
+// eslint-disable-next-line import/no-unresolved
+const pluginConfigFactory = require('@brainhubeu/gatsby-docs-kit/plugins');
+
+module.exports = {
+  siteMetadata: {
+    title: 'Brainhub template',
+    description: 'Default template for open-source libraries',
+    image: 'https://cdn-images-1.medium.com/max/1200/1*CLUFZFaXF6NG27NA3d_JkQ.jpeg',
+    url: 'https://screencloud.dev.brainhub.pl/',
```
Comment (piotr-s-brainhub) [inline example/gatsby-config.js:13]: what is this URL?
I get 404

File: README.md [hunk 6]
```diff
@@ -27,10 +27,10 @@ Gatsby Docs Kit is a set of tools, extending [Gatsby](https://www.gatsbyjs.org/)
 
 Ensure you have the latest version of [Node](https://nodejs.org/en/download/) installed. We also recommend you install [Yarn](https://yarnpkg.com/en/docs/install) as well.
 
-Clone starter repository:
+Clone repository:
 
 ```bash
-git clone git@github.com:brainhubeu/gatsby-docs-kit-starter.git docs-www && cd docs-www && rm -rf .git
+git clone git@github.com:brainhubeu/gatsby-docs-kit.git && cd example
```
Comment (piotr-s-brainhub) [inline README.md:33]:
```suggestion
git clone git@github.com:brainhubeu/gatsby-docs-kit.git && mv gatsby-docs-kit/example docs-www && rem -rf gatsby-docs-kit
```

File: tools/seed/my-first-docs.md (hunk 4 above)
Comment (RobertHebel) [reply tools/seed/my-first-docs.md:28]: Because this page doesn't exist

File: example/README.md (hunk 3 above)
Comment (RobertHebel) [reply example/README.md:4]: done

File: example/package.json (hunk 2 above)
Comment (RobertHebel) [reply example/package.json:26]: honestly idk, I just moved the repository 🤷‍♂️ 

I will move them to deps and I will do the same in docs-www

File: example/gatsby-config.js (hunk 5 above)
Comment (RobertHebel) [reply example/gatsby-config.js]: Probably someone left it here by the mistake. Changed.

File: example/gatsby-config.js [hunk 7]
```diff
@@ -0,0 +1,26 @@
+'use strict';
+
```
Comment (piotr-s-brainhub) [inline example/gatsby-config.js:2]:
```suggestion
```

Comment (piotr-s-brainhub) [reply example/gatsby-config.js:2]: I guess it's not needed
